jina/orchestrate/flow/base.py

```diff
@@ -1490,6 +1492,14 @@ def _get_address_table(self, address_table):
                     attrs='underline',
                 )
             )
+            address_table.append(
```

Comment:
check if the graphql endpoint is there right?

Reply:
I think it is always there when the gateway uses http, which we check for above